src/model/documentselection.js

```diff
@@ -601,6 +633,26 @@ class LiveSelection extends Selection {
 		}
 	}
 
+	overrideGravity( customRestore ) {
```

Comment:
Docs?

Reply:
`LiveSelecton` class is private and has no docs.
```
// `LiveSelection` is used internally by {@link module:engine/model/documentselection~DocumentSelection} and shouldn't be used directly.
```